src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/locationfinder/AppointmentLocationsFinderService.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.whereabouts.services.locationfinder
+
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.whereabouts.dto.prisonapi.ScheduledAppointmentDto
+
+@Service
+class AppointmentLocationsFinderService {
```

Comment:
This is a wrapper for AppointmentLocationsFinder.  Why? Because I wrote AppointmentLocationsFinder first and because I needed a service I could mock.  Not very strong reasons.  I'm tempted to merge the two classes.